docs/translations.md

```diff
@@ -0,0 +1,25 @@
+# Translations
+
+See also Database Schema
+
+To allow for internationalisation, the title and message text for a single notification is purposefully not stored in the database.
+
+Instead, a plugin or theme should store a key, which is linked to a translatable message in the plugin or theme's translation files.
+
+For example, given the following list of translatable strings in a plugin 
+
+```
+$notifications = array( 
+    'plugin_slug_new_podcast_title' => __('Some title', 'plugin-slug'), 
+    'plugin_slug_message' => __('Some message', 'plugin-slug'), 
+);
```

Comment:
WordPress i18n functions match translated strings with their source string (e.g.: 'Some title' ). We may be able to do the same to provide an easier API for plugin developers.

Note that this still holds the plugin responsible for returning the correct translation.

Reply:
> Note that this still holds the plugin responsible for returning the correct translation.

I think this is the correct implementation, but it doesn't require any database schema changes to be possible.